src/test/kotlin/interactor/GetBest3CitiesForBuyingSoftDrinksInteractorTest.kt

```diff
@@ -0,0 +1,82 @@
+package interactor
+
+import data.FakeDataSource
+import data.TestCase
+import model.CityEntity
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class GetBest3CitiesForBuyingSoftDrinksInteractorTest{
+
+
+    // region init
+    private lateinit var best3CitiesForBuyingSoftDrinks: GetBest3CitiesForBuyingSoftDrinksInteractor
+    private lateinit var fakeData: FakeDataSource
+
+    @BeforeAll
+    fun setup() {
+
+        fakeData = FakeDataSource()
+        best3CitiesForBuyingSoftDrinks =
+            GetBest3CitiesForBuyingSoftDrinksInteractor(fakeData)
+        fakeData.changeDataSource(TestCase.BestCityForSavingMoney)
+    }
+    // endregion
+
+    @Test
+    fun should_ReturnTrue_When_TheCityHasNotNullRequireValues() {
```

Comment:
move to ` fun `should return true when the city has not null require values`()` style.